doc/man1/openssl-pkcs12.pod.in

```diff
@@ -351,7 +367,7 @@ If none of the B<-clcerts>, B<-cacerts> or B<-nocerts> options are present
 then all certificates will be output in the order they appear in the input
 PKCS#12 files. There is no guarantee that the first certificate present is
 the one corresponding to the private key. Certain software which requires
-a private key and certificate and assumes the first certificate in the
+a private key and certificates and assumes the first certificate in the
```

Comment:
IMO this is wrong change although the original sentence is slightly awkward. Perhaps replace it with: `Certain software which tries to get a private key and the corresponding certificate might assume that the first certificate in the file is the one corresponding to the private key, but that may not always be the case.`